.typos.toml

```diff
@@ -1,31 +1,31 @@
-[default.extend-identifiers]
-AnserStyle = "AnserStyle"
-datas = "datas"
-impl_froms = "impl_froms"
-selfs = "selfs"
+[files]
+extend-exclude = [
+    "*.rast",
+    "bench_data/",
+    "crates/parser/test_data/lexer/err/*",
+]
+ignore-hidden = false
+
+[default]
+extend-ignore-re = [
+    # ignore string which contains $0, which is used widely in tests
+    ".*\\$0.*",
+    # ignore generated content like `boxed....nner()`, `Defaul...efault`
+    "\\w*\\.{3,4}\\w*",
+    '"flate2"',
+    'IDENT "SER"',
```

Comment:
Same for this one? And we could also ignore `crates/project-model/test_data/*`, I don't think there's anything useful in there.

And would you mind squashing at the end?

Reply:
As you wish.
Now comes the squashing ...